bash_scripts/bash_script/script.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+function dirsolder {
+        logger "dirs older"
+        IFS=$'\n'
+        read -p "Input dir's older than: " days
+        date=$(date +"%b %d %H:%M" -d "$days day ago")
+        for var in $(ls -ld ~/*/  | awk '{printf "%s %s %s %s\n", $6, $7, $8, $9}')
+        do
+                if [ "$date" \> "$var" ]
+                then
+                        dir_name=$(echo "$var" | awk '{print $NF}' | grep -oP '(?<=\/home\/anton\/).+')
+                        dir_creation=$(echo "$var" | awk '{print $1" "$2" "$3}')
+                        printf "$dir_name " ; tput setaf 1 ; echo "$dir_creation" ; tput setaf 7
+                fi
+        done
+}
+
+function filesolder {
+        logger "files older"
+        IFS=$'\n'
+        read -p "Input file's older than: " days
+        date=$(date +"%b %d %H:%M" -d "$days day ago")
+        for var in $(ls -l -p ~/ | egrep -v /$ | sed -n '1!p' | awk '{printf "%s %s %s %s\n", $6, $7, $8, $9}')
+        do
+                if [ "$date" \> "$var" ]
+                then
+                        file_name=$(echo "$var" | awk '{print $NF}')
+                        file_creation=$(echo "$var" | awk '{print $1" "$2" "$3}')
+                        printf "$file_name " ; tput setaf 2 ; echo "$file_creation" ; tput setaf 7
+                fi
+        done
+}
+
+function pcmonitoring {
+        logger "pc monitoring"
+        tput setaf 1 ; echo "Total PIDS: `top -bn1 | grep Tasks | awk '{printf $2}'`" ; tput setaf 7
+        tput setaf 2 ; echo "Total MEMORY usage: `free -m | awk 'NR==2{printf "%s/%s", $3,$2 }'` MB" ; tput setaf 7
+        tput setaf 3 ; echo "Total CPU usage: `top -bn1 | grep load | awk '{printf "%.2f", $(NF-2)}'` %" ; tput setaf 7
+        tput setaf 4 ; echo "Total DISC usage: `df -h | awk '$NF=="/"{printf "%d/%d (%s)\n", $3,$2,$5}'` GB" ; tput setaf 7
+}
+
+function addapptohosts {
+        logger "add app tohosts"
+        sudo grep -qxF '192.168.56.4 myapp.com' /etc/hosts || echo '192.168.56.4 myapp.com' >> /etc/hosts
+}
+
+function changeappinhosts {
```

Comment:
Function names should have a bit human-readable format: `change_app_in_hosts `

Reply:
rename functions